NEWS

```diff
@@ -1,3 +1,32 @@
+etcd v3.1.0 (2017-01-13)
+- faster linearizable reads (implements Raft read-index)
+- automatic leadership transfer when leader steps down
+- etcd uses default host IP if advertise URL is not given
+- cluster rejects removing members if quorum will be lost
+- SRV records (e.g., infra1.example.com) must match the discovery domain
+  (i.e., example.com) if no custom certificate authority is given
+  - TLSConfig ServerName is ignored if custom certificate is given
```

Comment:
TLSConfig ServerName is ignored with user-provided certificates for backwards compatibility; to be deprecated in 3.2